flamedisx/x1t_sr1.py

```diff
@@ -15,21 +19,126 @@
 ##
 # Yield maps
 ##
-
-
 s1_map, s2_map = [
     fd.InterpolatingMap(fd.get_resource(fd.pax_file(x)))
     for x in ('XENON1T_s1_xyz_ly_kr83m-SR1_pax-664_fdc-adcorrtpf.json',
               'XENON1T_s2_xy_ly_SR1_v2.2.json')]
 
+##
+# Parameters
+##
+def_g1 = 0.142
+def_g2 = 11.4 # don't divide by weird shits. will explode downstream.
```

Comment:
Comment is and not very enlightening (and not very polite)

Reply:
g2 bottom as defined in 1902.11297v2. instead of g2 total.